packages/aws-cdk/lib/init-templates/v1/app/csharp/cdk.template.json

```
@@ -1,3 +1,15 @@
 {
-  "app": "dotnet run -p src/%name.PascalCased%/%name.PascalCased%.csproj"
+  "app": "dotnet run -p src/%name.PascalCased%/%name.PascalCased%.csproj",
+  "watch": {
+    "include": ["**"],
+    "exclude": [
+      "README.md",
+      "cdk.json",
+      "src/%name.PascalCased%/obj",
```

This is better, but I have an even better idea 🙂:

```suggestion
      "src/*/obj",
```

What do you think about doing that instead @comcalvi?

---

I don't see why not, but I also don't see why. What's the advantage @skinny85?

---

1. It's shorter.
2. If you rename your app, you don't have to change these settings.